src/content/1.7/faq/i-need-help.md

```diff
@@ -0,0 +1,26 @@
+---
+title: I need help
+---
+
+# I need help
+
+If you have questions or need help, please do not open a GitHub issue as we use GitHub issues only to discuss about bugs and new features for the PrestaShop project.
+
+The very best place to start looking for an answer is:
+
+- Here (the Developer documentation), if you are a developer
+- The [User's guide](https://doc.prestashop.com/display/PS17/User+Guide) if you are a user
+
+{{% notice tip %}}
+If you are a developer and you think there is a missing item in the Developer documentation, consider [creating an issue on the Documentation Repository](https://github.com/PrestaShop/docs/issues). This will help us planning what to do to improve it.
```

Comment:
```suggestion
If you are a developer and you think there is a missing item in the Developer documentation, consider [creating an issue on the Documentation Repository](https://github.com/PrestaShop/docs/issues). This will help us discuss what to do to improve it.
```